### ISSUE TITLE
focus is lost when the chat model picker is escaped

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#247409]
1. focus the chat model picker via keyboard
2. hit escape
3. 🐛 focus is lost

It should be on the model picker. This works well for the mode picker.

**Comment by meganrogge** [comment]
This is working in insider's today.
